compliance_checker/cf/cf.py

```diff
@@ -2784,7 +2794,6 @@ def check_climatological_statistics(self, ds):
                 return ret_val
             # handle 1-d and 2d coordinate bounds
             if (clim_var.ndim + 1 != ds.variables[clim_var.climatology].ndim):
-                valid = False
```

Comment:
This seems unused. Is it OK to remove it @lukecampbell? (There are a few more below.)